packages/analytics/shippers/fullstory/src/fullstory_shipper.ts

```diff
@@ -73,26 +73,31 @@ export class FullStoryShipper implements IShipper {
     }
 
     // User-level context
-    if (version || cloudId) {
+    if (typeof isElasticCloudUser === 'boolean') {
```

Comment:
Why are we checking for the type, what else could it be?

Reply:
If `isElasticCloudUser` is `false`, we also want to report it. TBH, I could have gone with `typeof isElasticCloudUser !== 'undefined'`, but I guess I'm a positive person and didn't want to negate 😄 

I can change it to make it more permissive if you think it's necessary.

Reply:
I like your coding optimism :-D